ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1050,6 +1050,16 @@ port_channel_interfaces:
     esi: < EVPN Ethernet Segment Identifier (Type 1 format) >
     rt: < EVPN Route Target for ESI with format xx:xx:xx:xx:xx:xx >
     lacp_id: < LACP ID with format xxxx.xxxx.xxxx >
+    evpn_ethernet_segment:
+      redundancy: < Multihoming redundancy mode (all-active | single-active | default --> all-active) >
+      designated_forwarder_election:
+        algorithm: < modulus | preference | default --> modulus >
```

Comment:
This algorithm default value is eos specific. But AVD assumes no default. So please remove this default value.